trace-utils/src/trace_utils.rs

```diff
@@ -40,6 +45,210 @@ pub async fn get_traces_from_request_body(body: Body) -> anyhow::Result<(usize,
     Ok((size, traces))
 }
 
+#[inline]
+fn get_v5_strings_dict(reader: &mut Reader<impl Buf>) -> anyhow::Result<Vec<String>> {
+    let dict_size =
+        read_array_len(reader).map_err(|err| anyhow!("Error reading dict size: {err}"))?;
+    if dict_size as i32 > 25 * 1e6 as i32 {
+        anyhow::bail!(
+            "Error deserializing strings dictionary. Dict size is too large: {dict_size}"
+        );
+    }
+    let mut dict: Vec<String> = Vec::with_capacity(dict_size.try_into()?);
+    for _ in 0..dict_size {
+        let val: Value = read_value(reader)?;
+        match val {
+            Value::String(s) => {
+                let s = s.to_string().replace(&['\"'][..], "");
```

Comment:
This feels so weird. Are the strings escaped this way by the Python tracer, or is it the `rmpv` library that does this (which would be really strange). I'm pretty sure that the Java Tracer does not do this, but things like AppSec and CI Visibility would send escaped JSON in fields. It should at least have a comment.

Reply:
Yup fixed, just shouldn't use `to_string()` on the `Value::String`